eng/native/configurecompiler.cmake

```diff
@@ -549,6 +549,7 @@ if (CLR_CMAKE_HOST_UNIX)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${MACOS_VERSION_MIN_FLAGS} ${DISABLE_OVERRIDING_MIN_VERSION_ERROR}")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${MACOS_VERSION_MIN_FLAGS} ${DISABLE_OVERRIDING_MIN_VERSION_ERROR}")
     set(CMAKE_ASM_FLAGS "${CMAKE_ASM_FLAGS} ${MACOS_VERSION_MIN_FLAGS} ${DISABLE_OVERRIDING_MIN_VERSION_ERROR}")
+    set(CMAKE_OBJCXX_FLAGS "${CMAKE_C_FLAGS} ${MACOS_VERSION_MIN_FLAGS} ${DISABLE_OVERRIDING_MIN_VERSION_ERROR}")
```

Comment:
shouldn't this be `CMAKE_OBJC_FLAGS` ? we're not using Objective-C++ as far as I know

Reply:
Ah sorry, I just saw this is specifically for .mm files so those _are_ Objective-C++.

Though that raises the question if we should use `CMAKE_C_FLAGS` for `CMAKE_OBJC_FLAGS` and `CMAKE_CXX_FLAGS` for `CMAKE_OBJCXX_FLAGS`

Reply:
Ufff. That was cut & paste error... it should start with `set(CMAKE_OBJCXX_FLAGS "${CMAKE_OBJCXX_FLAGS}`

Reply:
ok, and what about CMAKE_OBJC_FLAGS?

Reply:
We can add that too, I guess.

Reply:
Done